projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -158,88 +210,138 @@ export class OwcJsonService {
+      return;
+    } else {
+      const isList = /,/g.test(values);
+      if (isList) {
+        // values: `${string},${string}`
+        const splitValues = values.split(',');
+        if (splitValues.length > 0) {
+          const parsed: Array<string | ILayerIntervalAndPeriod> = []; //
+          for (const value of splitValues) {
+            const parsedSingle = this.parseSingleTimeOrPeriod(value);
+            if (typeof parsedSingle === 'object' && parsedSingle.interval) {
+              if (!parsedSingle.periodicity && period) {
+                parsedSingle.periodicity = period;
+              } else if (!parsedSingle.periodicity && !period) {
+                console.warn(`Interval without a period`, values, period);
+              }
+            }
+            parsed.push(parsedSingle);
+          }
+          return parsed;
+        }
+      } else {
+        // `${string}/${string}` | `${string}/${string}/P${string}`
+        const parsedSingle = this.parseSingleTimeOrPeriod(values);
+        if (typeof parsedSingle === 'object' && parsedSingle.interval) {
+          if (!parsedSingle.periodicity && period) {
+            parsedSingle.periodicity = period;
+          } else if (!parsedSingle.periodicity && !period) {
+            console.warn(`Interval without a period`, values, period);
+          }
+          return parsedSingle;
+        } else if (typeof parsedSingle === 'string') {
+          return [parsedSingle];
+        }
       }
-      return outputs;
     }
-
-    const parsed = this.parseSingleTimeOrPeriod(input);
-    return parsed;
-
-  }
-
-  private parseSingleTime(timeString: string): DateTime {
-    return DateTime.fromISO(timeString);
-  }
-
-  private parsePeriod(periodString: string): Interval {
-    return Interval.fromISO(periodString);
   }
 
+  /**
+   * time could be:
+   *
+   * - date
+   * - start/end/duration //Geoserver specific
+   * - start/end
+   * - start/duration, and duration/end
+   */
   private parseSingleTimeOrPeriod(time: string): string | ILayerIntervalAndPeriod | null {
-    const dateTime = this.parseSingleTime(time);
+    const dateTime = DateTime.fromISO(time);
     if (dateTime.isValid) {
-      return dateTime.toISO();
-    }
-
-    const interval = this.parsePeriod(time);
-    if (interval.isValid) {
-      const period = this.parseISO8601Period(time);
-      const intervalObject: ILayerIntervalAndPeriod = {
-        periodicity: period,
-        interval: `${interval.start.toISO()}/${interval.end.toISO()}`
-      };
-      return intervalObject;
+      return dateTime.toUTC().toISO();
+    } else {
+      // is Interval ----------------------------
+      const interval = Interval.fromISO(time);
+      if (interval.isValid) {
+        const period = this.parseISO8601Period(time);
+        const intervalObject: ILayerIntervalAndPeriod = {
+          periodicity: period,
+          interval: `${interval.start.toUTC().toISO()}/${interval.end.toUTC().toISO()}`
```

Comment:
Here we used toUTC() so the time values are passed over like configured and not get rewritten.